src/configs/gemm-config.c

```diff
@@ -1741,11 +1741,11 @@ static void init_qp8_f32_qb4w_gemm_config(void) {
     if (XNN_ENABLE_ARM_I8MM && hardware_config->use_arm_neon_i8mm) {
       #if XNN_ENABLE_ARM_I8MM
         qp8_f32_qb4w_gemm_config.minmax.qp8gemm_bl[XNN_MR_TO_INDEX(1)] = xnn_init_hmp_qp8gemm_bl_ukernel(xnn_qp8_f32_qb4w_gemm_minmax_ukernel_1x4c16s2__aarch64_neondot);
-        qp8_f32_qb4w_gemm_config.minmax.qp8gemm_bl[XNN_MR_TO_INDEX(8)] = xnn_init_hmp_qp8gemm_bl_ukernel(xnn_qp8_f32_qb4w_gemm_minmax_ukernel_8x4c16s2__neoni8mm_mstep2);
+        qp8_f32_qb4w_gemm_config.minmax.qp8gemm_bl[XNN_MR_TO_INDEX(16)] = xnn_init_hmp_qp8gemm_bl_ukernel(xnn_qp8_f32_qb4w_gemm_minmax_ukernel_16x4c16s2__neoni8mm_mstep4);
```

Comment:
This is OK for now. 

For future, I am not sure if this will be the best tile size for the budget phones or slightly older phones with i8mm. Perhaps in the future we can use cpu specific tile size i.e. larger on Cortex-x cores and smaller on mid cores like we do in other hmp cases?